e2e/src/utils.ts

```diff
@@ -83,16 +86,30 @@ const events: Record<EventType, Set<string>> = {
   userDelete: new Set<string>(),
 };
 
-const callbacks: Record<string, () => void> = {};
+const idCallbacks: Record<string, () => void> = {};
+const countCallbacks: Record<string, { count: number; callback: () => void }> = {};
 
 const execPromise = promisify(exec);
 
 const onEvent = ({ event, id }: { event: EventType; id: string }) => {
-  events[event].add(id);
-  const callback = callbacks[id];
-  if (callback) {
-    callback();
-    delete callbacks[id];
+  // console.log(`Received event: ${event} [id=${id}]`);
```

Comment:
Is that intentional?

Reply:
Yeah it is good for debugging, but it's quite noisy